src/requirements.txt

```diff
@@ -3,33 +3,25 @@ black>=20.8b1
 bokeh==2.3.3
 chart_studio==1.1.0
 coverage==6.2
-cryptography==36.0.2
+cryptography==37.0.4
 decorator==5.1.1
 flake8>=3.8.4
 html5lib==1.1
 idna==3.3
 jinja2==3.0.3
-jsonschema==3.2.0
```

Comment:
if you look at the installation logs, a number of these deps are already installed by other packages. Removed the unneeded deps and updated a couple of packages that have newer versions available.